package.json

```
@@ -1,15 +1,15 @@
 {
-  "name": "@1inch/token-plugins",
+  "name": "@1inch/token-hooks",
   "version": "1.3.0",
```

```suggestion
  "version": "2.0.0",
```
Because breaking changes